src/components/app/App.tsx

```diff
@@ -1,30 +1,75 @@
 import Main from '../main/Main';
 
 import styles from './App.css';
+import commonStyles from '../../styles/commonStyles.css';
 
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { Routes, Route, BrowserRouter } from 'react-router-dom';
 
 import { Layout } from 'components/Layout';
 import { Countries } from 'components/countries/Countries';
 import { Details } from 'components/details/Details';
 import { Feedback } from 'components/feedback/Feedback';
 import { Page404 } from 'components/page404/Page404';
+import { setInLocalStorage, getFromLocalStorage } from 'services/localStorage';
 
 const App: FC = () => {
+  const [token, setToken] = useState<string>('');
+
+  async function getToken(user: string, password: string): Promise<string> {
+    const body = {
+      user,
+      password
+    };
+
+    const ENV = process.env.NODE_ENV === 'production' ? 'production' : 'development';
+
+    let URL =
+      ENV === 'development'
+        ? 'https://cors.bridged.cc/https://pfa.foreca.com/authorize/token?expire_hours=-1'
```

Comment:
please don't use 3rd party services as proxy

Reply:
Write my own server for resolve CORS mistake?

Reply:
yes